src/components/side_panel/chart/gauge_chart_panel/gauge_chart_design_panel.ts

```diff
@@ -67,13 +67,17 @@ interface PanelState {
 
 export class GaugeChartDesignPanel extends Component<Props, SpreadsheetChildEnv> {
   static template = "o-spreadsheet-GaugeChartDesignPanel";
-  static components = { ColorPicker };
+  static components = { ColorPickerWidget };
 
   private state: PanelState = useState({
     openedMenu: undefined,
     sectionRuleDispatchResult: undefined,
   });
 
+  setup() {
+    useExternalListener(window as any, "click", this.closeMenus);
```

Comment:
```suggestion
    useExternalListener(window, "click", this.closeMenus);
```
requires a test as well: :)